src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/util/AzureInstancePolicy.java

```diff
@@ -0,0 +1,64 @@
+package cloud.fogbow.ras.core.plugins.interoperability.azure.util;
+
+import cloud.fogbow.common.exceptions.InvalidParameterException;
+import cloud.fogbow.common.models.AzureUser;
+import cloud.fogbow.ras.constants.SystemConstants;
+import cloud.fogbow.ras.core.models.orders.ComputeOrder;
+import com.sun.istack.internal.Nullable;
```

Comment:
Change import to `javax.annotation.Nullable`

Reply:
Done.
I've made the change in the branch azure-compute-plugin-four-pr-fixes